conda_forge_tick/git_utils.py

```diff
@@ -308,63 +684,35 @@ def get_repo(
     repo : github3 repository
         The github3 repository object.
     """
-    gh = github3_client()
-    gh_username = gh.me().login
+    backend = github_backend()
+    feedstock_repo_name = feedstock_repo(fctx)
 
-    # first, let's grab the feedstock locally
-    upstream = feedstock_url(fctx=fctx, protocol=protocol)
-    origin = fork_url(upstream, gh_username)
-    feedstock_reponame = feedstock_repo(fctx=fctx)
+    try:
+        backend.fork("conda-forge", feedstock_repo_name)
+    except RepositoryNotFoundError:
+        logger.warning(f"Could not fork conda-forge/{feedstock_repo_name}")
+        with fctx.attrs["pr_info"] as pri:
+            pri["bad"] = f"{fctx.package_name}: does not match feedstock name\n"
```

Comment:
We need to make an issue to remind us to remove the package_name attribute from the feedstock context object and just use feedstock_name to avoid this.

Reply:
I am a bit confused now. I understand that
- `package_name` is the networkx node name of the feedstock
- Here, `feedstock_repo` uses the `feedstock_name` to generate the git URL
- So if the git URL does not exist, the value of `feedstock_name` is invalid.

Why is the error message "package_name does not match feedstock_name" then? (I just copied this from the existing code)

Are there any instances where `feedstock_name` is actually different from `package_name`? Or is this exactly the point you want to make?

Reply:
Yes this is the point I am making. Only feedstock_name matters. We should remove package_name all together. The names of the packages can differ from the feedstock, but bot's graph nodes use the feedstock_name.

Reply:
xref: https://github.com/regro/cf-scripts/pull/2660